checkov/arm/graph_builder/graph_to_tf_definitions.py

```diff
@@ -0,0 +1,13 @@
+from __future__ import annotations
+
+from pathlib import Path
+from typing import Any
+
+from checkov.arm.graph_builder.graph_components.blocks import ArmBlock
+
+
+def convert_graph_vertices_to_tf_definitions(
```

Comment:
please if you can and it does not affect the full flow, change the name it's not tf

Reply:
Done